sdk/storage/azure-storage-common/inc/azure/storage/common/reliable_stream.hpp

```diff
@@ -10,17 +10,19 @@
 
 namespace Azure { namespace Storage {
 
-  // options used by the fm callback that will get a bodyStream starting from last offset
-  struct HttpGetterInfo
-  {
-    int64_t Offset = 0;
-  };
+  namespace internal_ {
```

Comment:
The underscore comes before not after.
```suggestion
  namespace _internal {
```